orc8r/cloud/go/services/metricsd/prometheus/prometheus-cache/cache/cache_test.go

```diff
@@ -92,15 +103,25 @@ func TestScrape(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, http.StatusOK, rec.Code)
 
+	// since families are processed in an arbitrary order
+	// we can't check the string exactly. we can check
+	// how many lines it has. the rest of the tests check
+	// whether the metrics are sorted properly. there are 4 extra
+	// debug statements but 1 less newline in the output.
+	assert.Equal(t, strings.Count(sampleReceiveString, "\n")+4-1, strings.Count(rec.Body.String(), "\n"))
+
 	// parse the output to make sure it gives valid response
 	var parser expfmt.TextParser
 	parsedFamilies, err := parser.TextToMetricFamilies(rec.Body)
 	assert.NoError(t, err)
-	assert.Equal(t, 4, len(parsedFamilies))
+	assert.Equal(t, 5, len(parsedFamilies))
```

Comment:
can you also check that the actual metrics are what you expect?

Reply:
related to comment above, will do